EditImprovements/Edit.patch

```diff
@@ -0,0 +1,285 @@
+--- Edit.Mod	2022-01-28 22:03:24.791966000 +0100
++++ Edit.Mod	2022-01-28 21:56:05.478840000 +0100
+@@ -1,27 +1,20 @@
+ MODULE Edit; (*JG 2.11.90 / NW 28.11.2015*)
++
+   IMPORT Files, Fonts, Texts, Display, Viewers, Oberon, MenuViewers, TextFrames;
+ 
+   CONST CR = 0DX; maxlen = 32;
+     StandardMenu = "System.Close System.Copy System.Grow Edit.Search Edit.Store";
+ 
+   VAR W: Texts.Writer;
+-    time: LONGINT;
+-    M: INTEGER;
++    time, M: INTEGER;
```

Comment:
Yup, but LONGINT was replaced by INTEGER.

Reply:
As I said, aesthetics. In Oberon-07, `LONGINT` is a synonym to `INTEGER`.

Reply:
I regarded LONGINT as somehow deprecated. I expect it to be dropped sooner or later because Wirth doesn't like superfluous key words or identifiers. But maybe I'm wrong.

Reply:
While he did remove Edit.Replace from Project Oberon 2013, he did the original implementation on another system and it would have been easy for him to replace all LONGINT by INTEGER before even starting to port it.

And, in case it will disappear, it will have to disappear from all upsteam files as well. And all your LONGINT->INTEGER refactorings will create conflicts once we try to rebase our changes on the new upstream version.